src/apps/MASS3DPA-OMP.cpp

```diff
@@ -103,6 +103,20 @@ void MASS3DPA::runOpenMPVariant(VariantID vid) {
 
   case RAJA_OpenMP: {
 
+#if defined(RAJA_ENABLE_CUDA)
```

Comment:
A couple of things about this:

1. It still seems odd to me that we have to have GPU execution policies in code CPU variant files. Have you thought of giving the LaunchPolicy type a default template parameter in the second parameter slot, such as launch_undefined, which would make the first launch type the only valid option and attempting the second would generate a run-time error?
2. Since these policies are repeated in multiple files does it make sense to define them in a macro in the kernel header file?

Reply:
Hi Rich, yes we made an issue to rework Teams (in RAJA) to simplify this type of execution. Once I have that up in RAJA we can update it here. I also moved the policies to the header.